servers/curated-corpus-api/schema-shared.graphql

```diff
@@ -102,15 +102,16 @@ type Section {
   """
   sectionItems: [SectionItem!]!
   """
-  Optional description for the Section.
+  A short description of the Section. Relevant for custom sections.
```

Comment:
ML "daily briefing"  section will also have a description (subtitle) so I would omit "Relevant for custom sections" on this field